src/components/VerticalStepper/VerticalStepper.test.tsx

```diff
@@ -0,0 +1,130 @@
+import { fireEvent } from "@testing-library/dom";
+import VerticalStepper from "./VerticalStepper";
+import { renderCUI } from "@/utils/test-utils";
+interface Props {
+  activeIndex?: number;
+  completed?: Array<number>;
+  showItems?: Array<number>;
+}
+const label1Click = jest.fn();
+const label2Click = jest.fn();
+const label3Click = jest.fn();
+const label4Click = jest.fn();
+describe("VerticalStepper", () => {
+  const renderVerticalStepper = ({
+    activeIndex = 1,
+    completed = [],
+    showItems = [],
+  }: Props) =>
+    renderCUI(
```

Comment:
not fully related, but we should just name this render. We already know we are in the CUI

Reply:
it was to distinguish it from `render` of the react testing library